lib/Net/SAML2/Binding/Redirect.pm

```diff
@@ -262,6 +262,7 @@ sub verify {
 
     $sig = decode_base64($sig);
 
+    my $verified = 0;
     my %errors;
     foreach my $crt (@{$self->cert}) {
```

Comment:
This should move to `_verify($signed)` or something similar.
We can than do:
```
if (!$self->_verified) {
    croak/confess/die "Could not verify XML due to invalid signature(s)";
}
# Do things because it was verified
```

Reply:
`sub _verified($signed)` to be more precise :)